config/wdio/wdio.conf.js

```diff
@@ -11,7 +12,17 @@ const visualRegressionConfig = require('./visualRegressionConf');
 const TerraWDIOSpecReporter = require('../../reporters/wdio/TerraWDIOSpecReporter');
 
 /* Use to pass your host's IP when running wdio tests from a VM or behind a proxy. */
-const ip = process.env.WDIO_EXTERNAL_HOST || localIP.address();
+let ip = process.env.WDIO_EXTERNAL_HOST || localIP.address();
+const networkInterfaces = os.networkInterfaces();
+if (!process.env.WDIO_EXTERNAL_HOST) {
+  const utuns = [];
+  Object.keys(networkInterfaces).map(e => (e.includes('utun') ? utuns.push(e) : null));
+  utuns.forEach((utun) => {
+    if (networkInterfaces[utun] && networkInterfaces[utun][0] && networkInterfaces[utun][0].family === 'IPv4') {
+      ip = networkInterfaces[utun][0].address;
+    }
+  });
+}
```

Comment:
```suggestion
if (!process.env.WDIO_EXTERNAL_HOST) {
  const utun = Object.entries(networkInterfaces).find(([key, interface]) => key.includes('utun') && interface[0] && interface[0].family === 'IPv4')[1];

  if (utun) {
    ip = utun[0].address;
  }
}
```
Consider this alternate way of finding the address that will stop with the first ipv4 address instead of the last.